src/components/organisms/Sections/About/about.scss

```diff
@@ -3,7 +3,7 @@
     background-color: #f6ede0;
     padding-left: 15%;
     padding-right: 15%;
-
+    
```

Comment:
whitespace?

Reply:
Just am improved bootstrap container for desktop devices. Like in any other section, yes.

Reply:
...wtf